image-customize

```diff
@@ -231,6 +239,8 @@ def main():
     # options
     parser.add_argument('--base-image',
                         help='Base image name, if "image" does not match a standard Cockpit VM image name')
+    parser.add_argument('--fresh', action='store_true',
+                        help="Remove an existing overlay; by default, multiple image-customize calls are additive")
```

Comment:
help="Start fresh from the base image; ..."

Reply:
Done, thanks!